### ISSUE TITLE
[assn: damage control] Nerf the pump shotgun. It's pretty ridiculous.

### ISSUE THREAD

**Issue by Killibrew** [InZidiuZ/op-framework-issue-tracker#362]
**Name:**
Titan#3369

**Summary:**
The pump shotgun can one shot someone with full armor from 50 yards away. It's absolutely ridiculous. There is no gun fight it's just whoever has the pump shotgun is walking away from that encounter. 

**Reason:**
It would add to roleplay/gunfights when it comes down to it. Rolling up on a shooting scene as a police officer and immediately getting downed by one shot from a guy 50 yards away isn't that fun.

**Comment by ExpDev07** [comment]
I understand it's not fun. But weapons are dangerous. Wouldn't it be better to just tread and approach dangerous situations with more caution (using your car as a shield, going in with backup, etc)?

**Comment by Killibrew** [comment]
Oh yea I learned after the first 50 yard blast to respect it but like I said I was 50 yards away which shouldn't be a killshot from a shotgun. Unless these are slugs in there I guess. I just looked that up and they have a 75 yard effective range. (Keep in mind I'm just guessing these ranges in game) 

But it's the roleplay of it. If someone has a shotgun and just runs a bit and turns around and one shots you from 50 yards away there isn't anything to that. We had a guy outnumbered 3 to 1 and we were using cover and everything and he just one shot everyone from a ridiculous range. One of our guys even using the SMG. I don't know if the criminal side can get access to any better guns so that might be the best weapon against an SMG to even the playing field.
